test/libethereum/test/libethereum/BlockChain.cpp

```diff
@@ -537,6 +537,7 @@ BOOST_AUTO_TEST_CASE(rescue)
 
 			try
 			{
+				std::chrono::seconds(10); //try wait for block verification before rescue
```

Comment:
This will not make the process to wait 10s. It will just create an object representing 10 second interval and destroy it just after created.